src/main/kotlin/uk/gov/justice/digital/assessments/services/AssessmentService.kt

```diff
@@ -252,7 +252,7 @@ class AssessmentService(
   }
 
   private fun matchAnswers(
-    episodeAnswer: Map.Entry<String, List<String>>,
+    episodeAnswer: Map.Entry<String, List<Any>>,
```

Comment:
It's a shame Kotlin doesnt have `union`(?) types (AFAIK) - in TypeScript you would have something like
```
type AnswerValue = String | GPDetails | EmergencyContactDetails | PersonalContactDetails
```
Which would help remove the need for the `Any` type here